README.md

```diff
@@ -114,4 +103,16 @@ MONGO_USERNAME=*******
 MONGO_PASS=******
 MONGO_HOST=YOURHOST.mongodb.net
 DEFAULT_DATABASE=db
-```
\ No newline at end of file
+```
+
+## GitHub OAuth
+This project is now fully integrated with GitHub OAuth, please set the variables in  your .env 
+```bash
+GITHUB_CLIENTID=
+GITHUB_CLIENTSECRET=
+OAUTHLIB_INSECURE_TRANSPORT=true
+```
+And you will need to add the userID that will have writer access in the file Auth\userpy:
```

Comment:
Thanks for the clear instructions! 😄 

Worth noting here that if you are expecting the writer id to vary, it could be worth switching it to an environment variable